CMakeLists.txt

```diff
@@ -166,7 +166,7 @@ else()
   if(WITH_ZSTD)
     find_package(zstd REQUIRED)
     add_definitions(-DZSTD)
-    include_directories(${ZSTD_INCLUDE_DIR})
+    include_directories(${ZSTD_INCLUDE_DIRS})
```

Comment:
I think this also needs fixing in `cmake/modules/Findzstd.cmake` where it seems to be in mixed-case.